crates/matrix-sdk-ui/src/room_list/state.rs

```diff
@@ -89,7 +90,8 @@ impl Action for AddVisibleRoomsList {
             .add_list(
                 SlidingSyncList::builder(VISIBLE_ROOMS_LIST_NAME)
                     .sync_mode(SlidingSyncMode::new_selective())
-                    .timeline_limit(20),
+                    .timeline_limit(20)
+                    .required_state(vec![(StateEventType::RoomEncryption, "".to_owned())]),
```

Comment:
Should we also enable `m.room.member` events here, to help address https://github.com/matrix-org/matrix-rust-sdk/issues/2004?

Reply:
Maybe! @poljar, @stefanceriu, @manuroe, any thoughts?

Reply:
Isn't damir talking only about the rooms you're subscribing to and not the full lists? 🤔

Reply:
Yes, you're only sending events in a room that you're subscribed to. So only there we'll need the member events.